superset/cli/importexport.py

```diff
@@ -133,12 +133,14 @@ def export_datasources(datasource_file: Optional[str] = None) -> None:
     @click.option(
         "--path",
         "-p",
+        required=True,
```

Comment:
Changes to this file are not directly related to this PR, but seemed like an easy improvement while I ran manual testing.